doc/chapters/api/index.rst

```diff
@@ -32,102 +32,187 @@ BindMount
 Binds a path on the host to the container. Any missing parent directories in
 the path will be created.
 
-:Parameters:
-        :containerID: ``int32`` The ID obtained by CreateContainer method.
-        :pathInHost: ``string`` absolute path in the host.
-        :pathInContainer: ``string`` the absolute path to mount to in the container.
-        :readOnly: ``bool`` indicates whether the directory is read-only or not.
-
-|
+Parameters
+##########
+* containerID: ``int32`` The ID obtained by CreateContainer method.
+* pathInHost: ``string`` absolute path in the host.
+* pathInContainer: ``string`` the absolute path to mount to in the container.
+* readOnly: ``bool`` indicates whether the directory is read-only or not.
+
+Error sources
```

Comment:
The reason bad state has a colon is because there is only one possible issue with the container state, whereas with the path there are many things that could go wrong. So it is not a special case, it's just that having a single sub-item made little sense to me. But I can change it for consistency.